Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx

```diff
@@ -15,908 +15,907 @@
-  add(e2d, 717, 107, 856, 5, 1);
-  add(e2d, 717, 104, 856, 7, 0);
-  add(e2d, 717, 103, 856, 7, 1);
-  add(e2d, 717, 1033, 856, 0, 0);
-  add(e2d, 717, 1032, 856, 0, 1);
-  add(e2d, 717, 1031, 856, 0, 2);
-  add(e2d, 717, 1030, 856, 0, 3);
-  add(e2d, 717, 1133, 856, 2, 0);
-  add(e2d, 717, 1134, 856, 2, 1);
-  add(e2d, 717, 1135, 856, 2, 2);
-  add(e2d, 717, 1129, 856, 4, 0);
-  add(e2d, 717, 1130, 856, 4, 1);
-  add(e2d, 717, 1125, 856, 6, 0);
-  add(e2d, 717, 1126, 856, 6, 1);
-  add(e2d, 717, 308, 857, 1, 0);
-  add(e2d, 717, 307, 857, 1, 1);
-  add(e2d, 717, 306, 857, 1, 2);
-  add(e2d, 717, 305, 857, 1, 3);
-  add(e2d, 717, 304, 857, 1, 4);
-  add(e2d, 717, 211, 857, 3, 0);
-  add(e2d, 717, 212, 857, 3, 1);
-  add(e2d, 717, 213, 857, 3, 2);
-  add(e2d, 717, 214, 857, 3, 3);
-  add(e2d, 717, 215, 857, 3, 4);
-  add(e2d, 717, 206, 857, 5, 0);
-  add(e2d, 717, 207, 857, 5, 1);
-  add(e2d, 717, 208, 857, 5, 2);
-  add(e2d, 717, 201, 857, 7, 0);
-  add(e2d, 717, 202, 857, 7, 1);
-  add(e2d, 717, 203, 857, 7, 2);
-  add(e2d, 717, 1325, 857, 0, 0);
-  add(e2d, 717, 1326, 857, 0, 1);
-  add(e2d, 717, 1327, 857, 0, 2);
-  add(e2d, 717, 1243, 857, 2, 0);
-  add(e2d, 717, 1242, 857, 2, 1);
-  add(e2d, 717, 1241, 857, 2, 2);
-  add(e2d, 717, 1240, 857, 2, 3);
-  add(e2d, 717, 1234, 857, 4, 0);
-  add(e2d, 717, 1233, 857, 4, 1);
-  add(e2d, 717, 1229, 857, 6, 0);
-  add(e2d, 717, 1228, 857, 6, 1);
-  add(e2d, 718, 111, 858, 1, 0);
-  add(e2d, 718, 112, 858, 1, 1);
-  add(e2d, 718, 113, 858, 1, 2);
-  add(e2d, 718, 106, 858, 3, 0);
-  add(e2d, 718, 107, 858, 3, 1);
-  add(e2d, 718, 108, 858, 3, 2);
-  add(e2d, 718, 101, 858, 5, 0);
-  add(e2d, 718, 102, 858, 5, 1);
-  add(e2d, 718, 103, 858, 5, 2);
-  add(e2d, 718, 1138, 858, 0, 0);
-  add(e2d, 718, 1139, 858, 0, 1);
-  add(e2d, 718, 1133, 858, 2, 0);
-  add(e2d, 718, 1134, 858, 2, 1);
-  add(e2d, 718, 1128, 858, 4, 0);
-  add(e2d, 718, 1129, 858, 4, 1);
-  add(e2d, 718, 12, 859, 1, 0);
-  add(e2d, 718, 11, 859, 1, 1);
-  add(e2d, 718, 8, 859, 3, 0);
-  add(e2d, 718, 7, 859, 3, 1);
-  add(e2d, 718, 4, 859, 5, 0);
-  add(e2d, 718, 3, 859, 5, 1);
-  add(e2d, 718, 1033, 859, 0, 0);
-  add(e2d, 718, 1034, 859, 0, 1);
-  add(e2d, 718, 1029, 859, 2, 0);
-  add(e2d, 718, 1030, 859, 2, 1);
-  add(e2d, 718, 1025, 859, 4, 0);
-  add(e2d, 718, 1026, 859, 4, 1);
-  add(e2d, 719, 8, 860, 1, 0);
-  add(e2d, 719, 7, 860, 1, 1);
-  add(e2d, 719, 4, 860, 3, 0);
-  add(e2d, 719, 3, 860, 3, 1);
-  add(e2d, 719, 1029, 860, 5, 0);
-  add(e2d, 719, 1030, 860, 5, 1);
-  add(e2d, 719, 1025, 860, 7, 0);
-  add(e2d, 719, 1026, 860, 7, 1);
-  add(e2d, 719, 106, 860, 0, 0);
-  add(e2d, 719, 107, 860, 0, 1);
-  add(e2d, 719, 108, 860, 0, 2);
-  add(e2d, 719, 101, 860, 2, 0);
-  add(e2d, 719, 102, 860, 2, 1);
-  add(e2d, 719, 103, 860, 2, 2);
-  add(e2d, 719, 1134, 860, 4, 0);
-  add(e2d, 719, 1133, 860, 4, 1);
-  add(e2d, 719, 1129, 860, 6, 0);
-  add(e2d, 719, 1128, 860, 6, 1);
+  add(e2d, 707, 8, 720, 0, 0);
```

Comment:
Why this has changed? The new mapping looks correct while the old seems to be wrong, but why the old was not ok?